components/ai_chat/browser/ai_chat_api.cc

```diff
@@ -173,3 +173,7 @@ void AIChatAPI::QueryPrompt(
                                 headers, {});
   }
 }
+
+void AIChatAPI::ClearAllQueries() {
+  api_request_helper_.CancelAll();
+}
```

Comment:
I suppose this is fine for now, but when the AIChatAPI is owned by the Profile-based AI Chat Service, then this would cancel all queries for every conversation and not just the one being cleared.